### ISSUE TITLE
Mostrar que mensagem é "encaminhada"

### ISSUE THREAD

**Issue by laianeramos** [EvolutionAPI/evolution-api#709]
### Welcome!

- [X] Yes, I have searched for similar requests on [GitHub](https://github.com/code-chat-br/whatsapp-api/issues) and found none.

### What type of feature?

Functionality

### What is the motivation for the request?

É importante recebermos a informação de que o cliente/pessoa está enviando uma mensagem encaminhada, como acontece no whatsapp app/web, pois a falta dessa informação pode gerar ruídos e falhas de comunicação.

### Usage Examples

![image](https://github.com/user-attachments/assets/dd561fec-52e0-480b-a3a6-2ce052f05ee5)

### How should the feature be developed?

Mostrar um aviso igual quando uma mensagem é editada.
![image](https://github.com/user-attachments/assets/a2956c9c-dca6-4594-993c-2eca4fc205fc)

**Comment by dpaes** [comment]
@laianeramos creio que isso não depende só da evolution-api, porque para apresentar isso no chatwoot, ele tem que ter algo equivalente a isso já na aplicação. Se você fizer uma integração no chatwoot sem usar o evolution-api e tiver alguma flag ou algum mecanismo visual que mostre que foi encaminhado, dai os mantenedores do evolution-api podem fazer aparecer isso na integração, porém se no chatwoot não tem isso nativamente, dificilmente essa feature poderia ser implementada.

**Comment by laianeramos** [comment]
@dpaes sim, é claro que precisa ter no chatwoot uma forma de receber essa informação. Deixei a minha sugestão ali na issue: usar o mesmo recurso que já é usado pra mensagem editada.
